lib/fuser_mqubit.h

```diff
@@ -85,8 +89,19 @@ class MultiQubitGateFuser {
     unsigned visited;
```

Comment:
`GateF.visited` can be of type `Visited`, just to make that relationship explicit.

Reply:
Acknowledged.